pages/dist/index.html

```diff
@@ -62,10 +62,6 @@
         border-radius: 4px 4px 0 0;
       }
 
-      #controls {
```

Comment:
This element is no longer needed, as the controls and results panes have been consolidated.

Reply:
Consolidating the panes is really helpful, it will make it clearer to users which tool they're in.

Reply:
One counter point to having combined planes, is I feel the results are less clearly grouped under the "Results" header. I wonder if this could be subtly achieved by making the results a slightly different colour or an offset?

Reply:
Agreed, 'Results' and 'Typerighter' are the same weight and it's a bit strange. Honestly I'd favour just removing 'Typerighter' altogether for now – testers already know what they're in for – as I'm confident UX changes down the line will establish that context if it's still needed.

Reply:
I would be interested to see that, I think it's worth a try.